src/EventSubscriber/CacheableResponseSubscriber.php

```diff
@@ -21,14 +22,42 @@ class CacheableResponseSubscriber implements EventSubscriberInterface {
    */
   protected $logger;
 
+  /**
+   * Configuration Factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactory
+   */
+  protected $configFactory;
+
   /**
    * Constructs a new DefaultExceptionHtmlSubscriber.
    *
    * @param \Psr\Log\LoggerInterface $logger
    *   The logger service.
+   * @param \Drupal\Core\Config\ConfigFactory $configFactory
```

Comment:
You should really use the interface instead:

```suggestion
   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
```